nuxt.config.js

```diff
@@ -112,8 +111,20 @@ module.exports = {
     {src: '~/plugins/open-page-in-modal.js', ssr: false}
   ],
   modules: [
-    'cookie-universal-nuxt',
-    '@nuxtjs/dotenv'
+    ['@nuxtjs/dotenv', {
+      only: [
+        'WEBAPP_HOST',
+        'WEBAPP_PORT',
+        'WEBAPP_BASE_URL',
+        'API_HOST',
+        'API_PORT',
+        'EMBED_API_URL',
+        'SENTRY_DNS_PUBLIC',
+        'MAPBOX_TOKEN',
+        'MAINTENANCE',
```

Comment:
@appinteractive this will expose only public environment variables to the client.